index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="main"> 
+        <div class="header">
+            <header>
+                <nav>
+                    <div>
+                        <a>
+                            <img class="logo" src="./image/logo.png" alt="">
+                        </a>
+                    </div>
+                    <div class="navbar">
+                        <div></div>
+                        <div></div>
+                        <div></div>
+                    </div>
+                </nav>
+            </header>
+        </div>
+        <div class="eclipse-3">
+            <img src="./image/Ellipse3(1).png" alt="">
+        </div>
+        <div class="body">
+            <div class="name">
+                I’m Saikat Glad to see you!
+            </div>
+            <div class="description">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </div>
+            <div class="contact">
+                LETS CONNECT
+            </div>
+            <div class="connect-logo">
+                <div class="logo-div">
+                    <div>
+                        <img src="./image/twiter.svg" alt="">
+                    </div>
+                    <div>
+                        <img src="./image/M.svg" alt="">
+                    </div>
+                    <div>
+                        <img src="./image/Vector(3).svg" alt="">
+                    </div>
+                    <div>
+                        <img src="./image/Vector.png" alt="">
+                    </div>
+                    <div>
+                        <img src="./image/Vector(4).svg" alt="">
+                    </div>
+                    <div></div>
+                </div>  
```

Comment:
- [ ] As this is a list of your icons, I would advise you to use an unordered list <ul> tag to contain your icons as <li>
```
<ul>
    <li><img src="./image/Vector(4).svg" alt=""></li>
    <li><img src="./image/Vector.png" alt=""></li>
    <li><img src="./image/Vector(3).svg" alt=""></li>
</ul>
```

Reply:
Did the same.